test_fuzz_comps.py

```diff
@@ -236,6 +236,7 @@ def statements():
         st.from_type(ast.Global),
         st.from_type(ast.Expr),
         st.from_type(ast.FunctionDef),
+        st.from_type(ast.Name),
```

Comment:
This would just generate statements of the form `a`, right? Not sure why that's useful.

Reply:
In the simplified reproducer for https://github.com/python/cpython/issues/104602 a simple `b` after the comprehension would expose the bug. (In your form it was a `print(b)` but the print isn't necessary.) So it seems useful to allow these to be generated?

Reply:
That's true. One concern is that this will just throw NameError in most cases, making the run tests less useful. But it can still help catch scoping bugs.

Reply:
Although with only two allowed names, the chance of `NameError` decreases some? You may be right, I think we will need to observe some actual runs.